### DISCUSSION TITLE
图表控件中的饼图能否增加指引线？

### DISCUSSION THREAD

**Discussion by yangf85** [dotnetcore/BootstrapBlazor#3210] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

如图
![image](https://github.com/dotnetcore/BootstrapBlazor/assets/42733580/361af729-699f-4eda-a136-5f71d229695d)

**Comment by bb-auto** [comment]
@yangf85 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment]
@yangf85 这个需要去看看 `Chart.js` 是否支持
